### ISSUE TITLE
<pubPlace> vs <place>

### ISSUE THREAD

**Issue by duncdrum** [TEIC/TEI#2011]
so `<bibl>` seems to be the place for bibliographic references of not just books (movies, opera performances, and a few others in what i m currently working on). So far so good, however, I took `<pubPlace>` to be syntactic sugar for `<place type="publication">` yet that doesn't seem to be the case. 

This project wishes to record information about first performances and other type of places that usually go with citation style references of art works, yet `<place>` is not allowed as a direct child of `<bibl>` only `<pubPlace>`. I m aware that without much fuss i can do `<pubPlace type="first-performance">` but that strikes me as semantically wrong. I think to allow `<bibl>` to capture not just texts, it would make sense to allow `<place>` as a direct child analogues to how `<pubPlace>` already works

here is a sample of what i have in mind (currently invalid):

```xml
<bibl xml:id="W0631" type="artwork">
        <title xml:lang="en" type="main">When Will You Return?</title>
        <title xml:lang="zh" type="main">何日君再来</title>
        <author ref="#AG0582"/>
        <date when="1939"/>
        <place ref="#SP0436"/>
</bibl>
```

compared to (valid): 
```xml
<bibl xml:id="W0006" type="text">
        <title xml:lang="en" type="main">My Apprenticeship</title>
        <title xml:lang="ru" type="main">V lyudyakh</title>
        <title xml:lang="zh" type="main">在人间</title>
        <title xml:lang="zh-Latn-pinyin" type="main">Zai renjian</title>
        <author ref="#AG0107"/>
        <date when="1956"/>
        <pubPlace ref="#SP0005"/>
        <publisher>Renmin wenxue chubanshe</publisher>
</bibl>
```

**Comment by peterstadler** [comment]
`<pubPlace>` is more analogous to `<placeName type="publication">` (concerning its content model). Would that be an option to replace your proposed  `<place ref="#SP0436"/>` with `<placeName ref="#SP0436"/>`?

**Comment by duncdrum** [comment]
Hmm while better then `pubPlace` I still find this counterintuitive, the bibliography references a `place`, not its name. Similarly `author` references a `person` not their name.  So the following guidelines example is just three persName elements? 

```xml
<author>
 <persName>Beaumont, Francis</persName> and
 <persName>John Fletcher</persName>
</author>

```

Should I now include an `author` element alongside all the other `persName` variants in my prosopography section? What about publishers with multiple places of publication? Clearly these aren’t just variant names of the place of publication? 

If I m the only one who feels that this is odd I ll go ahead and close this issue. But also with respect to external authorities and LOD practices making these names instead of primary entities, doesn’t quite sit right with me.

**Comment by martindholmes** [comment]
@duncdrum I'm interested to see this:

"the bibliography references a place, not its name"

which caused me to think again about how bibliography items work. I've always thought of a bibl as containing a sequence of  referring strings, and the tags as identifying the roles of those strings; so:

`<pubPlace>London</pubPlace>`

is as Peter suggested a special case (descendant class, really, in my old object oriented brain) of `<placeName>` meaning the place name where publication took place (`<placeName>` itself being a special case of `<name>`, and `<name>` ultimately being a special case of `<rs>`). But if there is no string in there, then it can't be a referring string. In that case, I'm not sure that you have a real bibliography entry; rather you have a metadata record, which is a conceptually different kind of thing.

**Comment by duncdrum** [comment]
@martindholmes indeed I'm in the process of generating the central authority lists for the standoff element, not encoding a transcribed bibliography. Both of course would use `listBibl`. 

So from the discussion so far, i d say that the content model is needlessly ambiguous. I could live with `bibl` being a descendant class of `rs` but then the content model of e.g. `<author>` should reflect that by demanding string contents to be referenced (same for `publisher` `pubPlace`, etc) .  Also it is unclear if two `<author>` mean the collaboration of two persons or if the they denote spelling variants of the name of one person. 

on the other hand, i would prefer the primary entities that can go into the various `listXXX` elements to be cross reference able, allowing persons or institutions in `bibl` to be substituted by their full `person` or `org` encodings. This would allow for a more consistent use of xxxName elements inside authority lists, as they gather all meaningful name variants as such. In the current system it still appears inconsistent to not do, which is a) rather redundant, and b) shows that there is an ontological difference between the name string and the main entity:

```xml
 <person sex="M" xml:id="AG0002">
        <persName type="main" xml:lang="en"> <surname>Mou</surname> <forename>Dunbai</forename> </persName>
        <persName type="main" xml:lang="zh"> <surname>牟</surname> <forename>敦白</forename> </persName>
        <persName type="main" subtype="author" xml:lang="en"> <surname>Mou</surname> <forename>Dunbai</forename> </persName>
        <persName type="main" subtype="author" xml:lang="zh"> <surname>牟</surname> <forename>敦白</forename> </persName>
       ...
</person>
```

TLDR: I think that the elements of entity lists (`person`, `org`, `date`, `place`, etc) should consistently reference either string representations (names) or entities (things). If the former, the elements content model should demand string contents. I'd prefer they reference primary entities (`<date when="1999"/>`) not the referencing string. So `author` inside `bibl` can be substituted by `<person>` **not** `<name type="person" subtype="author">`

**Comment by martindholmes** [comment]
@duncdrum On this:

`<person type="author">`

I don't think I agree that an author is a type of person. I think authorship is a role played by a person with respect to a piece of work.

**Comment by duncdrum** [comment]
@martindholmes i was hasty with the demo, its adjusted now. switch `@type` for `@role`. In either case, i don't see being an author as a type of name.

**Comment by MegJBrown** [comment]
VF2F subgroup discussed this and thinks `<pubPlace>` would serve the purpose needed; if the concern is that a performance isn't publication, we recommend `<placeName>`, which is also allowed in`<bibl>`.
